ui/src/store/store.js

```diff
@@ -0,0 +1,65 @@
+import create from 'zustand';
+import { makeAgoricChainStorageWatcher } from '@agoric/rpc';
+
+export const useStore = create((set, get) => ({
+  watcher: makeAgoricChainStorageWatcher('http://localhost:26657', 'agoriclocal'),
+  brands: [],
+  brandToKeyword: {},
+  keywordToBrand: {},
+  assetBrand: {},
+  priceBrand: {},
+  brandToDisplayInfo: {},
+  smartWalletPurses: [],
+  vbankPurses: [],
+  wallet: null,
+  notifierState: { open: false, severity: '', message: '' },
+  exchangeAssets: [],
+  vbankAssets: [],
+  buyOrders: [],
+  sellOrders: [],
+  setOrders: (simpleExchange) => {
+    const buyOrders = [...simpleExchange.buys];
+    const sellOrders = [...simpleExchange.sells];
+    console.log('buyOrders: ', buyOrders);
+    console.log('sellOrders: ', sellOrders);
+    set({ buyOrders, sellOrders });
+  },
+  notifyUser: (severity, message) => {
+    set(() => ({
+      notifierState: { open: true, severity, message }
```

Comment:
I think this is for the snackbar. Do you think we should add it? @GrigoriGrigoryanBytepitch